test/rounding.jl

```diff
@@ -495,3 +495,11 @@ end
         end
     end
 end
+
+@testset "Rounding to Zero with RoundFromZero" begin
```

Comment:
I think this name is more accurate. I'd also include the issue number.
```suggestion
@testset "Rounding to floating point types with RoundFromZero #55820" begin
```

Reply:
I'd prefer if you did it, thank you